packages/renderer/src/lib/pod/PodEmptyScreen.svelte

```diff
@@ -10,8 +10,9 @@ const helloImage = 'quay.io/podman/hello:latest';
 const commandLine = `podman run -dt --pod new:${myFirstPod} ${helloImage}`;
 
 let inProgress = false;
+const provideInfosCopy = $providerInfos;
 
-$: selectedProviderConnection = $providerInfos
+$: selectedProviderConnection = provideInfosCopy
```

Comment:
here I'm not sure to follow why there is this change regarding promise ?

Reply:
Seems unrelated reverted by 7b3e7853b0dfb5515bfbe50164695c227632a782